src/main.js

```diff
@@ -0,0 +1,18 @@
+import { game, gameTools, ALERTS } from "./store.js";
+import {
+  chooseBoard,
+  gameMoves,
+  restart,
+  ReturnOneStep,
+  showRecord,
+  saveGame,
+  loadGame,
+} from "./handlers.js";
+
+chooseBoard(game, gameTools);
+gameMoves(game, gameTools);
+restart(game, gameTools);
+ReturnOneStep(game, ALERTS);
```

Comment:
why is it Capital case? in general we use Capital case for class (i.e. es6 classes) or for Components (like react component which is type of classes)....

Reply:
it's just a typo. I'll fix it